<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPW</class>
 <widget class="QWidget" name="GPW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPW - Client</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayoutConnection">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAdresseIP">
       <property name="text">
        <string>Adresse IP :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditAdresseIP">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEditPort">
       <property name="text">
        <string>1025</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButtonConnexion">
       <property name="text">
        <string>Connexion</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxGame">
     <property name="title">
      <string>Jeu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutGame">
      <item>
       <layout class="QGridLayout" name="gridLayoutGame">
        <item row="0" column="0">
         <widget class="QLabel" name="labelNomJoueur">
          <property name="text">
           <string>Nom du Joueur :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditNomJoueur"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelScore">
          <property name="text">
           <string>Score :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditScore"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonEnvoyerScore">
          <property name="text">
           <string>Envoyer Score</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonAfficherScore">
          <property name="text">
           <string>Afficher Scores</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPersonnage">
     <property name="title">
      <string>Personnage</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutPersonnage">
      <item>
       <widget class="QLabel" name="labelPersonnage">
        <property name="text">
         <string>Personnage attribué :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPersonnage">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxEtat">
     <property name="title">
      <string>État</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutEtat">
      <item>
       <widget class="QTextEdit" name="textEditEtat">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxScores">
     <property name="title">
      <string>Scores des Joueurs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutScores">
      <item>
       <widget class="QTextEdit" name="textEditScores">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
